packages/recorder-backend/src/recorder-server.ts

```diff
@@ -33,6 +34,7 @@ export class RecorderServer implements IRecorderServer {
         private httpPort: number = DEFAULT_RECORDER_HTTP_PORT,
         private wsPort: number = DEFAULT_RECORDER_WS_PORT,
         private transportInstance: ITransport = transport,
+        private testManager: TestManager = new TestManager('SW'),
```

Comment:
Looks like hardcode, that we don't want to have
Discuss it with @flops

Reply:
Yes, it was discussed, it's temporary

Reply:
It's ok for me as mock. The bach-end is my part mostly.